src/comps/recover.tsx

```diff
@@ -0,0 +1,233 @@
+        });
+        throw new Error("User not connected");
+      }
+
+      // ensure that the stx address and txId are present
+      if (!stxAddress || !txId) {
+        notify({
+          type: NotificationStatusType.ERROR,
+          message: "Invalid URL parameters",
+        });
+        throw new Error("User not connected");
+      }
+
+      const signersAggregatePubKey = (await getAggregateKey()).slice(2);
+
+      // Combine the version and hash into a single Uint8Array
+      const serializedAddress = serializeCVBytes(principalCV(stxAddress));
+
+      // get the publicKey from the user payment address
+      // user cannot continue if they're not connected
+      const paymentAddress = walletInfo.addresses.payment!;
+
+      const reclaimPublicKey = paymentAddress.publicKey;
+
+      // Parse lockTime from env variable
+      const parsedLockTime = parseInt(lockTime || "144");
+
+      // Create the reclaim script and convert to Buffer
+      const reclaimScript = Buffer.from(
+        createReclaimScript(parsedLockTime, reclaimPublicKey),
+      );
+
+      const reclaimScriptHex = uint8ArrayToHexString(reclaimScript);
+
+      const signerUint8Array = hexToUint8Array(signersAggregatePubKey!);
+
+      const depositScript = Buffer.from(
+        createDepositScript(signerUint8Array, maxFee, serializedAddress),
+      );
+      // convert buffer to hex
+      const depositScriptHexPreHash = uint8ArrayToHexString(depositScript);
+
+      const emilyReqPayload = {
+        bitcoinTxid: txId,
+        bitcoinTxOutputIndex: 0,
+        reclaimScript: reclaimScriptHex,
+        depositScript: depositScriptHexPreHash,
+      };
+
+      console.log({ emilyReqPayload });
+      //console.log({ emilyReqPayloadClient: JSON.stringify(emilyReqPayload) });
+
+      // make emily post request
+      const response = await fetch("/api/emilyDeposit", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(emilyReqPayload),
+      });
+
+      if (!response.ok) {
+        notify({
+          type: NotificationStatusType.ERROR,
+          message: `Issue with Request to Emily`,
+        });
+
+        throw new Error("Error with the request");
+      }
+      notify({
+        type: NotificationStatusType.SUCCESS,
+        message: `Request successful`,
+      });
+      setShowStepper(true);
+    } catch (err: any) {
+      throw new Error(err);
+    }
+  };
+
+  const renderTxId = () => {
+    const txId = searchParams.get("txId") || "";
+    if (txId.length > 10) {
+      return `${txId.slice(0, 5)}...${txId.slice(-5)}`;
+    }
+    return txId;
+  };
+  if (showStepper) {
+    return <RecoverReview txId={searchParams.get("txId") || ""} />;
+  }
+
+  return (
+    <>
+      <FlowContainer>
+        <>
+          <div className="w-full flex flex-row items-center justify-between">
+            <Heading>Recover Lost Transaction</Heading>
```

Comment:
Should we add some more details about what this is about?

Reply:
Added `In the rare situation where your transaction was lost, use this tool to ensure It's placed back on track within our system` - don't love it but a placeholder till coms team looks it over